src/utils/temp_folder.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import { cleanUpTempFolder, getTempFolder } from './temp_folder';
+import * as fs from 'fs';
+import * as os from 'os';
+
+describe('getTempFolder function', () => {
```

Comment:
please use multiple describes:
- describe: "temp folder functions"
  - describe: "getTempFolder function"
    - it: 'returns a folder that exists'
    - it: 'can be called twice in a row'
    - it: 'returns a folder that contains the correct subfolders'
  - describe: "cleanUpTempFolder function"
    - it: 'removes the temporary folder from the local system'
    - it: 'can be called twice in a row'